### PR TITLE
Atul assignment4 branch

### PR DESCRIPTION
**Pull request by AtulSAug19**

### CODE REVIEW

**Review by ghost — APPROVED** [review]
Code readability is good 
Keep it up!

File: src/AtulS/CharacterCount.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+//WAP to character occurring in user defined String and Character
+package AtulS;
+
+import java.util.Scanner;
+
+public class CharacterCount {
+
+	int findcharnew(String name, char ch, int index) {
```
Comment (ghost) [inline src/AtulS/CharacterCount.java:8]: why we are passing 3rd param as index.. what is the use of it..

**Review by ghost — APPROVED** [review]
Good Job Atul.. some of code is excellent..
Good, you are writing different function for each operation it clear your thought process more clear to  us.
Keep it up.
code is good to merge

File: src/AtulS/CharacterCount.java (hunk 1 above)
Comment (AtulSAug19) [reply src/AtulS/CharacterCount.java:8]: To achieve recursion in this program, we require one variable which will be compared with string length. Hence we can define index as instance variable or function parameter.
Please correct me if i am wrong.

Comment (ghost) [reply src/AtulS/CharacterCount.java:8]: Ok @AtulSAug19

**Review by ghost — APPROVED** [review]
Code Good to merge
keep practicing

**Pull request merged**